.env.test

```diff
@@ -0,0 +1,13 @@
+# Flask server configuration.
+FLASK_APP=todo_app/app
+FLASK_ENV=development
+
+# Change the following values for local development.
+SECRET_KEY=secret-key
+
+TRELLO_TOKEN=aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
+TRELLO_API_KEY=bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb
+TRELLO_BOARD_ID=000000000000000000000000
```

Comment:
I love clearly fake credentials - I certainly didn't have to worry about whether these are real! 😅

Reply:
My only concern was to make them the same length as the real credentials, just for a small amount of realism 😆